pandas/io/tests/parser/test_network.py

```diff
@@ -27,51 +34,15 @@ def setUp(self):
                          'pandas/io/tests/parser/data/salaries.csv')
 
     @tm.network
-    def test_url_gz(self):
-        url = self.base_url + '.gz'
-        url_table = read_table(url, compression="gzip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_bz2(self):
-        url = self.base_url + '.bz2'
-        url_table = read_table(url, compression="bz2", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_xz(self):
-        url = self.base_url + '.xz'
-        url_table = read_table(url, compression="zip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_zip(self):
-        url = self.base_url + '.zip'
-        url_table = read_table(url, compression="zip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
+    def test_compressed_urls(self):
```

Comment:
we run yielded test in computation and io/pickle

will be much easier when we switch to pytest